### ISSUE TITLE
Error: AttributeError: 'module' object has no attribute 'file'

### ISSUE THREAD

**Issue by xDroni** [tokland/youtube-upload#245]
Python 2.7.12
Linux Ubuntu 16.04

Error occurs every time I try to upload a video

```
Using client secrets: /root/YouTube-Tools/client-secrets.json
Using credentials file: /root/.youtube-upload-credentials.json
Traceback (most recent call last):
  File "/usr/local/bin/youtube-upload", line 10, in <module>
    main.run()
  File "/usr/local/lib/python2.7/dist-packages/youtube_upload/main.py", line 261, in run
    sys.exit(lib.catch_exceptions(EXIT_CODES, main, sys.argv[1:]))
  File "/usr/local/lib/python2.7/dist-packages/youtube_upload/lib.py", line 42, in catch_exceptions
    fun(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/youtube_upload/main.py", line 255, in main
    run_main(parser, options, args)
  File "/usr/local/lib/python2.7/dist-packages/youtube_upload/main.py", line 174, in run_main
    youtube = get_youtube_handler(options)
  File "/usr/local/lib/python2.7/dist-packages/youtube_upload/main.py", line 160, in get_youtube_handler
    get_code_callback=get_code_callback)
  File "/usr/local/lib/python2.7/dist-packages/youtube_upload/auth/__init__.py", line 38, in get_resource
    storage = oauth2client.file.Storage(credentials_file)
AttributeError: 'module' object has no attribute 'file'
```

**Comment by TJChiou** [comment]
same question here.

**Comment by zhangolve** [comment]
same question

**Comment by feacluster** [comment]
Same issue using Raspberry Pi 2.  I already tried "sudo pip install --upgrade google-api-python-client" and I am all up to date.

**Comment by feacluster** [comment]
I put "from oauth2client import file" in main.py and it worked!

**Comment by dawahnigeria** [comment]
same issue here
Linux Ubuntu 16.04

**Comment by MHRosenberg** [comment]
same issue also with Linux Ubuntu 16.04. Feacluster's solution above did not work for me.

**Comment by tabooki** [comment]
Same issue here.

**Comment by mrbesen** [comment]
Same issue with ubuntu 18.04, python 2.7.
i could solve that problem with the fix mentined by @feacluster 
after i found the ```main.py``` file and deleted the ```main.pyc``` file
my path of main.py: ```/usr/local/lib/python2.7/dist-packages/youtube_upload/main.py```

**Comment by mrbesen** [comment]
i just saw there is a fix here #273

**Comment by mrbesen** [comment]
and here #262 #264

**Comment by xavten** [comment]
I got the same issue but solution mentioned above doesn't work!

**Comment by RushilRobotics** [comment]
Same issue and placing: "from oauth2client import file"  in main.py doesnt help

**Comment by marttiku** [comment]
You have to build again after fixing the source. Run "sudo python setup.py install" in the project directory

**Comment by eaglegamma** [comment]
This did not work for me, even after rebuilding.

**Comment by kootsoop** [comment]
@eaglegamma What system are you using?  I just tried the "from oauth2client import file" fix and it seems to do the trick on Ubuntu 16.04, Python 2.7.12.

**Comment by kootsoop** [comment]
Applied fi

> I put "from oauth2client import file" in main.py and it worked!

@feacluster Thanks for this!  will push this update shortly.
